packages/core/src/components/workflow-step/workflow-step.tsx

```diff
@@ -79,33 +79,29 @@ export class WorkflowStep {
 
   @Watch('selected')
   selectedHandler() {
-    const selectedStyle = this.selected ? '--selected' : '';
-
-    if (this.status === 'open') {
-      this.iconName = this.selected ? 'circle-dot' : 'circle';
-      this.iconColor = `workflow-step-icon-default--color${selectedStyle}`;
-    }
-
-    if (this.status === 'done' && !this.disabled) {
```

Comment:
What about the disabled check? It is not tested in the refactored version anymore.